test/button/button-subtle-perceptual.js

```diff
@@ -0,0 +1,67 @@
+const puppeteer = require('puppeteer');
+const visualDiff = require('visual-diff');
+
+visualDiff.run((ctx) => {
+
+	describe('d2l-button-subtle', function() {
+
+		this.timeout(10000);
+		let browser, page;
+
+		before(async() => {
+			browser = await puppeteer.launch();
+			page = await browser.newPage();
+		});
+
+		after(() => browser.close());
+
+		const runTests = () => {
+
+			it('normal', async function() {
+				const rect = await ctx.puppeteer.getRect(page, '#normal');
+				await ctx.puppeteer.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
+			});
+
+			it('disabled', async function() {
+				const rect = await ctx.puppeteer.getRect(page, '#disabled');
+				await ctx.puppeteer.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
```

Comment:
The consumer can specify their own name, however `this.test.fullTitle()` works really well.